symplyphysics/laws/electricity/transmission_lines/ratio_of_average_power_to_incident_power_from_reflection_coefficient.py

```diff
@@ -28,9 +27,11 @@
 @validate_input(incident_power_=incident_power, average_power_=average_power)
 @validate_output(absolute_reflection_coefficient)
 def calculate_absolute_reflection_coefficient(incident_power_: Quantity, average_power_: Quantity) -> float:
+    if incident_power_.scale_factor < average_power_.scale_factor:
+        raise ValueError("The incident_power must be greater than the average power")
     result_expr = solve(law, absolute_reflection_coefficient, dict=True)[0][absolute_reflection_coefficient]
     result_expr = result_expr.subs({
         incident_power: incident_power_,
         average_power: average_power_,
     })
-    return convert_to_float(result_expr)
+    return result_expr
```

Comment:
Why this change?

Reply:
> Why this change?

Because I didn't understand what was meant by that remark:
https://github.com/blackyblack/symplyphysics/pull/716#discussion_r1580030552

I thought I should delete convert_to_float()

Reply:
You state that you've got absolute value, but solve() might return negative or positive solution. So I expect you do something with its sign or return just value, not absolute value.

Reply:
As soon as reflection coefficient is complex, its absolute value can be negative. So you shouldn't check the sign. Revert to `convert_to_float`.